package.json

```diff
@@ -93,6 +96,7 @@
   "homepage": "https://github.com/paldepind/snabbdom#readme",
   "husky": {
     "hooks": {
+      "commit-msg": "commitlint -E HUSKY_GIT_PARAMS",
```

Comment:
Taken right from [the local setup guide](https://commitlint.js.org/#/guides-local-setup).